applications/DamApplication/custom_constitutive/thermal_nonlocal_damage_3D_law.cpp

```diff
@@ -311,7 +311,7 @@ double&  ThermalNonlocalDamage3DLaw::CalculateNodalReferenceTemperature (const M
 
     for ( unsigned int j = 0; j < number_of_nodes; j++ )
     {
-      rNodalReferenceTemperature += ShapeFunctionsValues[j] * DomainGeometry[j].GetSolutionStepValue(NODAL_REFERENCE_TEMPERATURE);
+      rNodalReferenceTemperature += ShapeFunctionsValues[j] * DomainGeometry[j].GetSolutionStepValue(REFERENCE_TEMPERATURE);
```

Comment:
FastGet instead of Get

Reply:
Since this is inside the DamApp, I let them to change whatever they think it's convenient. :D

Reply:
Done!